Src/java/build.gradle

```diff
@@ -183,79 +185,89 @@ configure(subprojects.findAll {it.name in ['model', 'elm', 'quick', 'qdm', 'cql-
+                    name = project.name
+                    packaging = 'jar'
+                    description = "The ${project.name} library for the Clinical Quality Language Java reference implementation"
+                    url = 'http://cqframework.info'
 
                     scm {
-                        connection 'scm:git:git@github.com:cqframework/clinical_quality_language.git'
-                        developerConnection 'scm:git:git@github.com:cqframework/clinical_quality_language.git'
-                        url 'git@github.com:cqframework/clinical_quality_language.git'
+                        connection = 'scm:git:git@github.com:cqframework/clinical_quality_language.git'
+                        developerConnection = 'scm:git:git@github.com:cqframework/clinical_quality_language.git'
+                        url = 'git@github.com:cqframework/clinical_quality_language.git'
                     }
 
                     licenses {
                         license {
-                            name 'The Apache License, Version 2.0'
-                            url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+                            name ='The Apache License, Version 2.0'
+                            url = 'http://www.apache.org/licenses/LICENSE-2.0.txt'
                         }
                     }
 
                     developers {
                         developer {
-                            name 'Bryn Rhodes'
+                            name = 'Bryn Rhodes'
                         }
                         developer {
-                            name 'Chris Moesel'
+                            name = 'Chris Moesel'
                         }
                         developer {
-                            name 'Rob Dingwell'
+                            name = 'Rob Dingwell'
                         }
                         developer {
-                            name 'Jason Walonoski'
+                            name = 'Jason Walonoski'
                         }
                         developer {
-                            name 'Marc Hadley'
+                            name = 'Marc Hadley'
                         }
                     }
                 }
             }
         }
+        repositories {
+            maven {
+                credentials {
+                    username project.hasProperty("ossrhUsername") ? ossrhUsername : ""
+                    password project.hasProperty("ossrhPassword") ? ossrhPassword : ""
+                }
+
+                /* Use these to test locally (but don't forget to comment out others!)
+                def releasesRepoUrl = "file://${buildDir}/repo"
+                def snapshotsRepoUrl = "file://${buildDir}/ssRepo"
+                */
+
+                // change URLs to point to your repos, e.g. http://my.org/repo
+                def releasesRepoUrl = "file://${buildDir}/repo" //"https://oss.sonatype.org/service/local/staging/deploy/maven2/"
```

Comment:
This works for me locally if I just use the commented out URLs, so go ahead and uncomment them and commit that to the PR and I'll merge it